src/glmfit.jl

```diff
@@ -230,6 +240,43 @@ fit{M<:AbstractGLM}(::Type{M}, X::@compat(Union{Matrix,SparseMatrixCSC}), y::Abs
                     d::UnivariateDistribution, l::Link=canonicallink(d); kwargs...) =
     fit(M, float(X), float(y), d, l; kwargs...)
 
+"""
+    glm(X, y, family, link; wts, offset, dofit)
+
+Fits the generalized linear model, specified by the symbolic description of the
+model along with the family of distribution and a link function.
+
+#### Arguments:
+* `X::Matrix`: The formula which is the symbolic representation of the model to fit.
+Uses column symbols from the DataFrame data, for example, if names(data)
+=[:Y,:X1,:X2], then a valid formula is Y~X1+X2.
+* `y::AbstractVector`: Response vector.
+* `family::UnivariateDistribution`: Specifies the choice of variance, can be `Bernoulli()`, `Binomial()`, `Gamma()`, `Normal()` or `Poisson()`.
+* `link::Link=canonicallink(d)`: This function provides the link function for example, LogitLink() is a valid link for the Binomial() family.
```

Comment:
"This function provides" isn't useful here. Also, instead of an example, please list all supported links.

Finally, lines must be wrapped at 92 chars.